battleGame/Intermediate/Config/CoalescedSourceConfigs/Game.ini

```diff
@@ -1,5 +1,5 @@
 [Internationalization]
-LocalizationPaths=D:/UnrealProj/battlegame/battleGame/Content/Localization/Game
+LocalizationPaths=../../../../../../Users/Kevin/Documents/battleGame/battleGame/Content/Localization/Game
```

Comment:
Could we git ignore any of those?
On Jul 31, 2015 6:11 PM, "brad" <notifications@github.com> wrote:

> In battleGame/Intermediate/Config/CoalescedSourceConfigs/Game.ini
> <https://github.com/bradAnderson58/battleGame/pull/1#discussion_r36021335>
> :
>
> > @@ -1,5 +1,5 @@
> >  [Internationalization]
> > -LocalizationPaths=D:/UnrealProj/battlegame/battleGame/Content/Localization/Game
> > +LocalizationPaths=../../../../../../Users/Kevin/Documents/battleGame/battleGame/Content/Localization/Game
>
> Same issue
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bradAnderson58/battleGame/pull/1/files#r36021335>.
>